### ISSUE TITLE
Unable to create PR

### ISSUE THREAD

**Issue by lychichem** [microsoft/vscode-pull-request-github#2490]
- Extension version: 0.23.1
- VSCode Version: 1.53.2 x64 system install
- OS: Windows 10 x64 Pro 19042.804

Steps to Reproduce:

1. Checkout lychichem/winget-pkgs
2. add microsoft/winget-pkgs as upstream remote by `git remote add upstream git@github.com:microsoft/winget-pkgs.git`
2. Create a new branch from upstream/master and edit a manifest
3. Try to submit a pr
4. I cannot selete owner and branch, everything is blank.

Besides, extension version 0.22.0 just works fine.

**Comment by lychichem** [comment]
And, when I fail to create PR in one branch, checkout to another branch will even cause the extension show nothing (even the create pr icon).

**Comment by RMacfarlane** [comment]
When the view is blank, can you run the command `Developer: Open Webview Developer Tools` and report any errors in the console? Do you see any error notifications?

**Comment by lychichem** [comment]
> When the view is blank, can you run the command `Developer: Open Webview Developer Tools` and report any errors in the console? Do you see any error notifications?
> 
> Do you mean the entire view is blank, or the dropdowns are blank?

![屏幕截图 2021-02-13 082622](https://user-images.githubusercontent.com/9316590/107835731-7bf44600-6dd5-11eb-8b60-cef367a08355.png)
Here I checked out my repo and switch to zotero branch, all looks ok;

![屏幕截图 2021-02-13 082700](https://user-images.githubusercontent.com/9316590/107835771-9dedc880-6dd5-11eb-9c21-27d1820c3490.png)
Tried to make a PR, but it don't let me select repo and branch, just say "nothing to compare"

![屏幕截图 2021-02-13 082756](https://user-images.githubusercontent.com/9316590/107835814-bfe74b00-6dd5-11eb-8c09-194225cbc721.png)
Then I switched to another branch, but the view fails to switch.

**Comment by lychichem** [comment]
Is there anyone who can help me? I still cannot create PRs using 0.23.1, so I need to stay at 0.22.0

**Comment by RMacfarlane** [comment]
@lychichem When the blank "Create Pull Request" view is open, can you run the command "Developer: Open Webview Developer Tools" and copy and output that is in the console?

**Comment by lychichem** [comment]
@RMacfarlane So I created a branch `pr-test` in my repo and tried to make a pull request to microsoft/winget-pkgs, when I tried to make a pr, the extension do not list branches and forks. I however tried to make a pr and console show this:
VM97:22 Error: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=TypeError%3A%20Cannot%20read%20property%20'owner'%20of%20undefined&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at Ol (<anonymous>:22:52954)
    at <anonymous>:22:57748
    at $o (<anonymous>:22:66592)
    at _i (<anonymous>:22:106355)
    at wu (<anonymous>:22:90018)
    at yu (<anonymous>:22:89943)
    at su (<anonymous>:22:87291)
    at <anonymous>:22:45729
    at t.unstable_runWithPriority (<anonymous>:30:3462)
    at qa (<anonymous>:22:45438)
mi @ VM97:22
VM97:22 Uncaught Error: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=TypeError%3A%20Cannot%20read%20property%20'owner'%20of%20undefined&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at Ol (<anonymous>:22:52954)
    at <anonymous>:22:57748
    at $o (<anonymous>:22:66592)
    at _i (<anonymous>:22:106355)
    at wu (<anonymous>:22:90018)
    at yu (<anonymous>:22:89943)
    at su (<anonymous>:22:87291)
    at <anonymous>:22:45729
    at t.unstable_runWithPriority (<anonymous>:30:3462)
    at qa (<anonymous>:22:45438)

**Comment by lychichem** [comment]
Don't know why but I uninstalled the extension, reinstalled it and opened a repo that definitely can use this extension, tried to create a PR, then everything works ok. Then I opened this repo and checked out "master" from upstream and first tried to make a pr from master to my branch, then I can make PR from my branch to upstream. So maybe the reason is my fork do not have a "master" or "main" branch, but your extension requires the first PR you try to make must from "master" or "main". Anyway all works okay now. Closing this issue.
